.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: trusty
+
+language: python
+
+python: "3.6"
+
+cache: pip
+
+install: "pip install -r requirements.txt"
+
+script:
+  - make html
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_TOKEN
+  keep-history: true
+  local-dir: build/html
```

Comment:
This is a lot easier than I thought it would be!

Does the deploy target automatically copy build/html to the target-branch? that's pretty handy.

Let's give it a try!

Reply:
Yes. It just took a while to search around and find all the necessary pieces. We could have executed the commands in `build_gh_pages.sh`, but it turns out that good folks already implemented all the necessary features like this.